### ISSUE TITLE
Walker Learning Reward Function With Preferences

### ISSUE THREAD

**Issue by Stephanehk** [HumanCompatibleAI/imitation#685]
## Problem
Description of what problem this feature request aims to solve.
It would be great if we could get some hyperparameters for learning a reward function from preferences in the Walker2d environment (as well as the other Mujoco environments). I have tried out several different options across different seeds but cannot get anything to work. In general, it does not seem like the agent is learning even after 10^6 environment steps. If someone has any initial parameters that may work let me know. 

By the way, here are some of the parameters I tried out, which resulted in no improvement during training. I am not using any pre-training or disagreement sampling:

SyntheticGatherer
- temperature = 1

BasicRewardTrainer
- epochs = 5
- batch_size = 128
- lr=0.0003

PPO Agent
- 3 hidden layers of size 256
- n_steps=2048,
- batch_size=64,
- learning_rate=0.00005,
- n_epochs=20,
- gae_lambda = 0.92,

PreferenceComparisons
- fragment_length = 50
- num_iterations = 30
- initial_comparison_frac = 0.1
- total_timesteps=20000000
- total_comparisons = 3000
